vispy/geometry/meshdata.py

```diff
@@ -196,6 +201,59 @@ def get_vertices(self, indexed=None):
         else:
             raise Exception("Invalid indexing mode. Accepts: None, 'faces'")
 
+    def set_texcoords(self, texcoords=None, indexed=None):
+        """Set the mesh texture coordinates
+
+        Parameters
+        ----------
+        texcoords : ndarray | None
+            The array (Nv, 2) of vertex coordinates.
+        indexed : str | None
+            If indexed=='faces', then the data must have shape (Nt, 3, 2) and
+            is assumed to be already indexed as a list of faces.
+        """
+        if indexed is None:
+            if texcoords is not None:
+                self._texcoords = texcoords
+            self._texcoords_indexed_by_faces = None
+        elif indexed == 'faces':
+            self._texcoords = None
+            if texcoords is not None:
+                self._texcoords_indexed_by_faces = texcoords
+        else:
+            raise Exception("Invalid indexing mode. Accepts: None, 'faces'")
+
+    def get_texcoords(self, indexed=None):
+        """Get the texture coordinates
+
+        Parameters
+        ----------
+        indexed : str | None
+            If Note, return an array (N,2) of the texture coordinates of
+            vertices in the mesh. By default, each unique vertex appears only
+            once. If indexed is 'faces', then the array will instead contain
+            three vertices per face in the mesh (and a single vertex may appear
+            more than once in the array).
+
+        Returns
+        -------
+        texcoords : ndarray
+            The texture coordinates.
+        """
+        if indexed is None:
+            if (self._texcoords is None and
+                    self._texcoords_indexed_by_faces is not None):
+                self._compute_unindexed_texcoords()
+            return self._texcoords
+        elif indexed == 'faces':
```

Comment:
This was copied over from get_vertices().
Should there be support for texcoords indexed by faces? (In which case _compute_unindexed_texcoords() can be implemented as in the  case of vertices.)